src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+    private static final double POSITIVE_VALUE = 21;
+    private static final double NEGATIVE_VALUE = -21;
+    private static final double ZERO_VALUE = 0;
+    private static final double MAX_VALUE = Double.MAX_VALUE;
+    private static final double MIN_VALUE = Double.MIN_VALUE;
+    private static Calculator calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @AfterAll
+    static void afterAll() {
```

Comment:
```suggestion
    public static void afterAll() {
```
why @AfterAll? isn't it just another test?